contracts/data/EnumerableSet.sol

```diff
@@ -150,6 +150,57 @@ library EnumerableSet {
         return _remove(set._inner, bytes32(value));
     }
 
+    function toArray(Bytes32Set storage set)
+        internal
+        view
+        returns (bytes32[] memory)
+    {
+        uint256 len = _length(set._inner);
+        bytes32[] memory arr = new bytes32[](len);
+
+        unchecked {
+            for (uint256 index = 0; index < len; index++) {
+                arr[index] = at(set, index);
+            }
+        }
+
+        return arr;
+    }
+
+    function toArray(AddressSet storage set)
+        internal
+        view
+        returns (address[] memory)
+    {
+        uint256 len = _length(set._inner);
+        address[] memory arr = new address[](len);
+
+        unchecked {
+            for (uint256 index = 0; index < len; index++) {
+                arr[index] = at(set, index);
+            }
+        }
+
+        return arr;
+    }
+
+    function toArray(UintSet storage set)
+        internal
+        view
+        returns (uint256[] memory)
+    {
+        uint256 len = _length(set._inner);
+        uint256[] memory arr = new uint256[](len);
+
+        unchecked {
+            for (uint256 index = 0; index < len; index++) {
```

Comment:
Should look like:
```
for (uint256 index; index < len;) {
      arr[index] = at(set, index);
      ++i;
}
```

No need to initialize the index.

Reply:
Same applies for other for loops.

Reply:
https://github.com/solidstate-network/solidstate-solidity/pull/144/commits/f83e3986fb7d6d767a7130e61e0eef3842296c15